### DISCUSSION TITLE
Ypso MyLife CamAp XF and diasend

### DISCUSSION THREAD

**Discussion by michaellack** [NightscoutFoundation/xDrip#2508] [Q&A]
Hi, my son has just today been moved onto the Ypso MyLife CamAp XF system today after using the Insight pump for 7 years, we have had Dexcom G6 for 4 years using xdrip to follow him as it is much better than Dexcom follow and the alarms actually wake us up. His new system no longer uses Dexcom severs so we can't follow him anymore, we just get text messages every 30 minutes from CamAps. Is there anyway to pull the data out of diasend the same way or similar that it did from Dexcom? Many thanks in advance for any help you could provide, Michael.

**Comment by Navid200** [comment]
One option is to install xDrip on his phone and set it up to use the Companion App:  https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html
This will allow xDrip to show his readings from the CamAPS notifications on his phone.

Now, set up Nightscout and set his phone to upload to Nightscout: 
https://navid200.github.io/xDrip/docs/Nightscout_page

Now, set up your phone to follow Nightscout.

**Reply by michaellack** [reply]
Brilliant I will give that a try after school tomorrow. Thanks again, Michael.

Quick update, used the companion and set up the nightscout sever with the google cloud thing and all is working, you guys are brilliant, thank you so much.

**Reply by jamorham** [reply]
Please also try the current alpha channel version and enable the companion bluetooth setting - this should then support backfilling data.

**Reply by michaellack** [reply]
Thanks I will have a look at that as well. For your information, nearly one week since changing and Xdrip working brilliantly thanks again, the automatic text messages from the CamAPS FX sytem not so much, very erratic, either every 5 minutes but mostly none at all.

**Reply by MariaYpso** [reply]
And what about Mylife app? Can I send data to xdrip from Mylife app (without CamAPS)?

**Reply by michaellack** [reply]
Are you using Dexcom G6/G7?

**Reply by MariaYpso** [reply]
No, I am using Libre2 (LibreLink patched +xdrip, or sometimes Diabox+xdrip)

**Reply by michaellack** [reply]
Hi, I am sorry I have no experience with Libre and I would not want to tell you the wrong thing. I would ask jamorham or Navid200 as they are the experts.

**Reply by MariaYpso** [reply]
I will waiting the answer, thnx a lot!!!!
